test/codegen-generated-classes-test/src/test/java/software/amazon/awssdk/services/AsyncSignerOverrideTest.java

```diff
@@ -57,4 +58,10 @@ public void test_signerOverriddenForStreamingInput_takesPrecedence() {
 
         verify(mockSigner).sign(any(SdkHttpFullRequest.class), any(ExecutionAttributes.class));
     }
+
+    // TODO(sra-identity-and-auth): Add test for SRA way of overriding signer to assert that overridden signer is used.
+    //  To do this, need ability to inject AuthScheme which uses mock HttpSigner. This is pending https://i.amazon.com/SMITHY-1450
```

Comment:
removethe URL

Reply:
Removing in https://github.com/aws/aws-sdk-java-v2/pull/4574